test/util/namespaces/namespaces.go

```diff
@@ -20,6 +20,13 @@ import (
 // Test is the namespace to be use for testing
 const Test = "sriov-conformance-testing"
 
+var inhibitSecurityAdmissionLabels = map[string]string{
+	"pod-security.kubernetes.io/audit":               "privileged",
```

Comment:
We might want to align this with what we have for https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/371

Reply:
From what I see the labels have a delta.

Reply:
@wizhaoredhat this is fine for the test-pods we have this in other places where we run tests like https://github.com/openshift-kni/cnf-features-deploy/pull/1238

Reply:
Ah ok, got it. Thanks!